common/config/rush/pnpm-lock.yaml

```diff
@@ -8705,6 +8721,7 @@ packages:
       '@rollup/plugin-json': 4.1.0_rollup@1.32.1
       '@rollup/plugin-multi-entry': 3.0.1_rollup@1.32.1
       '@rollup/plugin-node-resolve': 8.4.0_rollup@1.32.1
+      '@rollup/plugin-replace': 2.4.2_rollup@1.32.1
```

Comment:
core-amqp does have a dev dependency on `@rollup/plugin-replace`
https://github.com/Azure/azure-sdk-for-js/blob/55f3461c46ec48f32d28921b0b6fd655adefdd07/sdk/core/core-amqp/package.json#L96